.gitignore

```diff
@@ -1,5 +1,8 @@
 build
-out
 .vscode
 build.log
-src/version_gen.rc
\ No newline at end of file
+src/version_gen.rc
+
+# outputs
+out*

```

Comment:
>* [](http://example.com/codeflow?start=3&length=1)

(nit) do we need this *? Seems like an overkill

Reply:
Fixed